tmpl/confirm_email.mustache

```diff
@@ -26,7 +26,7 @@
         <form id="form" action="/confirm" method="get">
           <input type="hidden" name="session" value="{{ session_id }}">
           <div class="entry">
-            <input type="text" name="code" maxlength="20" autofocus>
+            <input type="text" name="code" maxlength="20" autofocus autocomplete="nope" autocorrect="off" autocapitalize="off">
```

Comment:
I believe we can use `autocomplete="off"` here, since we don't have any password forms. It's the attribute value Google uses for their own 2FA screen. Otherwise, r+ with that change.